example/src/App.tsx

```diff
@@ -1,34 +1,51 @@
-import React, { useState } from 'react';
+import React from 'react';
 
-import { StyleSheet, View, Text } from 'react-native';
+import { StyleSheet, View, Button } from 'react-native';
 import Chargebee from '@chargebee/react-native-chargebee';
 
 export default function App() {
-  const [configComplete, isConfigComplete] = useState(false);
   const site = 'site';
-  const apiKey = 'test_key';
-  const sdkKey = 'sdk-key';
+  const apiKey = 'apiKey';
+  const sdkKey = 'sdkKey';
 
   React.useEffect(() => {
-    Chargebee.configure({
-      site: site,
-      publishableApiKey: apiKey,
-      sdkKey: sdkKey,
-    });
-    isConfigComplete(true);
+    configure(site, apiKey, sdkKey);
   }, []);
 
   return (
     <View style={styles.container}>
-      {!configComplete ? (
-        <Text>Pending config</Text>
-      ) : (
-        <Text>Config complete</Text>
-      )}
+      <Button
+        title="Configure"
+        onPress={() => configure(site, apiKey, sdkKey)}
+      />
+      <Button
+        title="Retrieve Product Identifers"
+        onPress={retrieveProductIdentifiers}
+      />
     </View>
   );
 }
 
+const retrieveProductIdentifiers = () => {
+  const queryParams = new Map<string, string>();
+  queryParams.set('limit', '100');
+  Chargebee.retrieveProductIdentifiers(queryParams)
```

Comment:
Can we use `async/await` here (and throughout) instead of `.then` for better readability?

Reply:
@cb-sriramthiagarajan Updated this.